applications/dashboard/src/scripts/entries/admin.tsx

```diff
@@ -34,9 +35,11 @@ applySharedPortalContext(props => {
     }, [setNavHeight]);
     return (
         <AppContext variablesOnly errorComponent={ErrorPage}>
-            <ScrollOffsetContext.Provider value={{ ...SCROLL_OFFSET_DEFAULTS, scrollOffset: navHeight }}>
-                {props.children}
-            </ScrollOffsetContext.Provider>
+            <BannerContextProviderNoHistory>
+                <ScrollOffsetContext.Provider value={{ ...SCROLL_OFFSET_DEFAULTS, scrollOffset: navHeight }}>
+                    {props.children}
+                </ScrollOffsetContext.Provider>
```

Comment:
I don't think this change was supposed to stay here once stuff was moved to the AppContext